docs/source/library.rst

```diff
@@ -4,13 +4,23 @@ torch.library
 .. currentmodule:: torch.library
 
 torch.library is a collection of APIs for extending PyTorch's core library
-of operators. It contains utilities for creating new custom operators as
-well as extending operators defined with PyTorch's C++ operator
+of operators. It contains utilities for testing custom operators, creating new
+custom operators, and extending operators defined with PyTorch's C++ operator
 registration APIs (e.g. aten operators).
 
 For a detailed guide on effectively using these APIs, please see
 `this gdoc <https://docs.google.com/document/d/1W--T6wz8IY8fOI0Vm8BF44PdBgs283QvpelJZWieQWQ/edit>`_
 
+Testing custom ops
+------------------
+
+Use :func:`torch.library.opcheck` to test custom ops for incorrect usage of the
```

Comment:
Why not `torch.testing.custom_op.opcheck` ?

Reply:
It's probably too many namespaces for people to keep track of? The analogy is torch.autograd.gradcheck -- would we have wanted to put it into torch.testing.autograd.gradcheck?

Reply:
torch.testing didn't exist when gradcheck was added?
I'm happy to open the discussion to move gradcheck there if you think that we should do a larger effort.

Reply:
Resolved offline: we'll go with torch.library.opcheck for now, but we're happy to put it into torch.testing too